build-config.json

```diff
@@ -10,7 +10,7 @@
     "srcStyleEntry": "./src/assets/style.scss",
     "srcStylesAny": "./src/assets/**/*.scss",
     "vendorAssets": ["./node_modules/highlight.js/styles/atom-one-dark.css"],
-    "invaderTagDefsBase": "./lib/invader/src/tag/hek/definition/"
+    "invaderDefsDir": "./lib/invader/src/tag/hek/definition/"
```

Comment:
I wanted to get away from calling the invader stuff "tag defs" because it's mainly oriented around structures which represent the tags or are part of them (e.g. data types for blocks). I don't want maintainers to confuse the invader struct defs with the tag defs built into c20.